### PR TITLE
Us 3 Task 37 Changed Spanish to English

### PR DESCRIPTION
**Pull request by mtjohn16**

### CODE REVIEW

**Comment by slaughter550** [comment]
This PR requires one more step of defining the english back to the spanish translation so work isn't lost. Please see "https://github.com/ser316asu/Heiligensee_SER316/blob/4fde213d5dae9dd1cb7374b97b9e6f5edd4f034a/src/net/sf/memoranda/util/Local.java#L31" on how to configure the `Local` class to pull messages in the default local

**Comment by mtjohn16** [comment]
Added the translation back to the properties file.

File: src/net/sf/memoranda/ui/AgendaPanel.java [hunk 1]
```diff
@@ -190,20 +190,25 @@ else if (d.startsWith("memoranda:tasks")) {
 						 }
 						 refresh(CurrentDate.get());
 					}else if (d.startsWith("memoranda:exportstickerst")) {
-						 /*  Falta agregar el exportar sticker mientras tanto..*/
+						 /*  Falta agregar el exportar sticker mientras tanto..
+						  * 	in english..."Missing adding export sticker in the meantime"*/
 						 final JFrame parent = new JFrame();
-						 String name = JOptionPane.showInputDialog(parent,Local.getString("Ingrese nombre de archivo a exportar"),null);
+						 //mtjohn16: changed spanish to english
+						 String name = JOptionPane.showInputDialog(parent,Local.getString("Enter file name to export"),null);
```
Comment (slaughter550) [inline src/net/sf/memoranda/ui/AgendaPanel.java]: For example, this string needs to be defined in the local `es` file as 

```
Enter file name to export = Ingrese nombre de archivo a exportar
```

**Comment by slaughter550** [comment]
@mtjohn16 the file only has to have new translations for the strings you're changing in this PR

**Comment by mtjohn16** [comment]
Sorry for the confusion.  Added the translations again.

**Comment by slaughter550** [comment]
I think you're only missing the definition for `Enter file name to export` now

**Comment by mtjohn16** [comment]
`Enter file name to export` was present, but i had left some characters off.  I double checked the other entries as well.

File: src/net/sf/memoranda/util/localmessages/messages_es.properties [hunk 2]
```diff
@@ -91,15 +92,21 @@ ENABLE SOUND NOTIFICATIONS = Activar notificaciones de sonido
 ENABLE SYSTEM TRAY ICON = Habilitar icono de sistema
 ENCODING = Codificación
 END DATE = Fecha de fin
+ENTER FILENAME TO EXPORT = Ingrese nombre de archivo a exportar
```
Comment (slaughter550) [inline src/net/sf/memoranda/util/localmessages/messages_es.properties]: This needs to have a space in `file name` to match the text above which is `Enter file name to export`

**Pull request merged**